test/function/samples/import-meta-url-compact/_config.js

```diff
@@ -0,0 +1,20 @@
+const assert = require('assert');
+const path = require('path');
+const URL = require('url-parse');
```

Comment:
url-parse needs to be added to rollup's dev dependencies. This is why tests on CI have been failing for some time.